virt/virt-2-4-release-notes.adoc

```diff
@@ -102,6 +102,10 @@ from being inadvertently started before an upload has completed.
 == Known issues
 //https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
 //https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
+* If you upgrade to {VirtProductName}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, use the newer version of the template. 
+
+two types of common templates are created for . For
```

Comment:
I think this line must be here by mistake?

Reply:
Line was deleted.